example_b2c/lib/main.dart

```diff
@@ -61,6 +61,14 @@ class _MyHomePageState extends State<MyHomePage> {
   final AadOAuth oauthB2Ca = AadOAuth(configB2Ca);
   final AadOAuth oauthB2Cb = AadOAuth(configB2Cb);
 
+  @override
+  void initState() {
+    super.initState();
+    oauthB2Ca.init();
```

Comment:
I think this should even be called in the main function which can be async (because initState cannot be async).

Then you could wait on init to complete.